docker-plugin/src/main/java/com/nirima/jenkins/plugins/docker/DockerJobProperty.java

```diff
@@ -14,6 +15,12 @@
 public class DockerJobProperty extends JobProperty<AbstractProject<?, ?>> {
 
     /**
+     * Image repository name for committed containers (image names)
+     */
+    @DataBoundSetter
+    public String imageRepository;
```

Comment:
Using public fields is bad solution, because you will unable to change it's behaviour later.
Constructor makes sense when there are not many parameters. Template for example has so many fields, so it better to use setters. Also constructor with critical fields + setter is more groovy friendly in case you will create some configuration programmatically.